DigitalLearningSolutions.Data/Services/RegistrationService.cs

```diff
@@ -23,7 +26,9 @@ public interface IRegistrationService
 
         string RegisterDelegateByCentre(DelegateRegistrationModel delegateRegistrationModel, string baseUrl);
 
-        void RegisterCentreManager(RegistrationModel registrationModel);
+        void RegisterCentreManager(AdminRegistrationModel registrationModel, int jobGroupId);

```

Comment:
Because it isn't needed to register an admin, it's needed to register a delegate for that admin. We should perhaps not be using the same models for all levels of the system? Maybe have models that the controllers pass to the services that are different from the models that are used to call the data service?